server/db/043-add-number-of-alerts-to-sessions.sql

```diff
@@ -0,0 +1,37 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 43;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE sessions
+        ADD COLUMN IF NOT EXISTS number_of_alerts INT;
+
+	-- Assume only 1 alert was sent for all sessions before this column was added
+	UPDATE sessions
+	SET number_of_alerts = 1
+	WHERE number_of_alerts IS NULL;
```

Comment:
One more comment DB-related comment... 
We're using `num_button_presses` at the column name for the equivalent thing on Buttons. When the DBs are combined, we probably want a name that is the same for both. I think that your `number_of_alerts` is fine, but if you can think of any reason why that doesn't make sense in the Buttons context, then it would be good to change it.

Reply:
Oo, really good point! I think `number_of_alerts` sounds good for both Sensors and Buttons, especially considering they both use the same code for the Brave **alert**er. So yay; that's ready for the merge!

Reply:
I think we talked a bit about this shared DB schema doc at one point, right? https://docs.google.com/document/d/1k-xkWnR7xoWRi6gEiN8p-nda5bhxDvOqNYCtPNi7OwY/edit

Once we're sure of this name, it can be updated to include this new column instead of `num_button_presses`.

Reply:
Wow; this document is pretty outdated. I think I'm going to start working on a page in the Living Doc that serves the same purpose, but with the changes I'll implement during the database merge.